gramex/apps/capture/chromecapture.js

```diff
@@ -74,6 +76,15 @@ function templatize(input) {
     .join('')
 }
 
+const browser_setup = async (args) => {
```

Comment:
Since this function is used only once, you can write it inline. No need for a separate function.